File: Keepass.Transfer.Android/Keepass.Transfer.Plugin/Receivers/TransferAccessReceiver.cs
using System.Collections.Generic;
using Android.App;
using Android.Content;
using Keepass2android.Pluginsdk;

namespace Keepass.Transfer.Plugin.Receivers
{
    [BroadcastReceiver(Enabled = true, Exported = true)]
    [IntentFilter(new[]{
        Strings.ActionTriggerRequestAccess,
        Strings.ActionReceiveAccess,
        Strings.ActionRevokeAccess
    })]
    public class TransferAccessReceiver : PluginAccessBroadcastReceiver
    {
        public override IList<string> Scopes => new List<string>
        {
            Strings.ScopeCurrentEntry,
            Strings.ScopeQueryCredentials
        };
    }
}

File: clients/quick/android/src/de/skycoder42/kpt/TransferAccessReceiver.java
package de.skycoder42.kpt;

import java.util.ArrayList;

import keepass2android.pluginsdk.PluginAccessBroadcastReceiver;
import keepass2android.pluginsdk.Strings;

public class TransferAccessReceiver extends PluginAccessBroadcastReceiver {
	@Override
	public ArrayList<String> getScopes() {
		ArrayList<String> scopes = new ArrayList<String>();
		scopes.add(Strings.SCOPE_CURRENT_ENTRY);
		scopes.add(Strings.SCOPE_QUERY_CREDENTIALS);
		return scopes;
	}
}
